docs/ballerina-to-oas/spec/spec.md

```diff
@@ -1214,6 +1216,51 @@ oneOf:
 >       type: string
 >   additionalProperties: false
 > ```
+> 2. If we have a record defined in a separate package, for example, packageA:
+> ```ballerina
+> type RecA record {|
+>    int a = 10;
+>    string c = "c";
+>|};
+>
+> ```
+> and it is included in packageB:
+>```ballerina
+> type RecB record {|
+>    *packageA:RecA;
+>     string b = "c";
+>|};
+>```
+> In the OpenAPI Specification (OAS) mapping for `RecB`, the default value access API cannot retrieve default values
+> from another package. To address this, we generate all the fields explicitly for `RecB` to ensure accessibility.
+> ```yaml
+> RecA:
+>   type: object
```

Comment:
Since we can't map the default values, we should have the `required` as well right?

Reply:
Yes, let me check. It seems there was an issue with our previous mapping. Currently, in version 2201.10.0, we declare it as not required (perhaps because we now provide a warning instead). I will check and try to fix it.

Reply:
Create an issue for tracking the https://github.com/ballerina-platform/ballerina-library/issues/7415